k8s/sandbox/cluster-00/jenkins-test/jenkins-pvc.yaml

```diff
@@ -0,0 +1,32 @@
+
+apiVersion: "v1"
+kind: "PersistentVolume"
+metadata:
+  name: "jenkinspv"
+spec:
+  capacity:
+    storage: "20Gi"
+  accessModes:
+    - "ReadWriteOnce"
+  storageClassName: managed-premium
+  persistentVolumeReclaimPolicy: Retain
+  azureDisk:
+    kind: Managed
+    diskName: test-disk
+    diskURI: /subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/resourceGroups/aks-infra-sbox-rg/providers/Microsoft.Compute/disks/test-disk
```

Comment:
was this preprovisioned or does creating it through yaml mean it will provision it itself?

Reply:
pre-provisioned

Reply:
manually for a test or is there a pipeline somewhere?

Reply:
I guess for prod-like envs we'll need a pipeline.  It will need to create the disk and output/store the resource ID

Reply:
manually for a test ATM